server/resources/migrations/68_attrs_etype_label_schema.up.sql

```diff
@@ -0,0 +1,44 @@
+ALTER TABLE attrs ADD COLUMN etype text;
+ALTER TABLE attrs ADD COLUMN label text;
+ALTER TABLE attrs ADD COLUMN reverse_etype text;
+ALTER TABLE attrs ADD COLUMN reverse_label text;
+
+ALTER TABLE attrs
+ADD CONSTRAINT attrs_etype_label_unique UNIQUE (app_id, etype, label);
+
+ALTER TABLE attrs
+ADD CONSTRAINT attrs_reverse_etype_label_unique UNIQUE (app_id, reverse_etype, reverse_label);
+
+CREATE OR REPLACE FUNCTION check_attrs_unique_names ()
+  RETURNS TRIGGER
+  AS $$
+BEGIN
+  IF EXISTS (
+    SELECT 1
+      FROM attrs
+     WHERE (app_id, reverse_etype, reverse_label) = (NEW.app_id, NEW.etype, NEW.label)
+  ) THEN
+    RAISE EXCEPTION 'trigger violation trg_attrs_unique_names'
+       USING DETAIL = format('Attr with (app_id, reverse_etype, reverse_label) (%s, %s, %s) already exist', NEW.app_id, NEW.etype, NEW.label),
```

Comment:
```suggestion
       USING DETAIL = format('Key (app_id, reverse_etype, reverse_label)=(%s, %s, %s) already exists', NEW.app_id, NEW.etype, NEW.label),
```

To make the parser work (https://github.com/instantdb/instant/blob/main/server/src/instant/util/exception.clj#L169)

I still think we should set `ERRCODE = '23505'` to make it easier to dispatch to the proper error handler in `exception.clj`

Reply:
Done